webknossos/examples/learned_segmenter.py

```diff
@@ -6,18 +6,23 @@
 from skimage.future import TrainableSegmenter
 
 import webknossos as wk
+from webknossos.dataset import SEGMENTATION_CATEGORY
 
+# We are going to use a public demo annotation for this example
```

Comment:
@jstriebel @fm3 This does not work. The annotation can not be downloaded. Server returns `401 - NotAuthorized`  error.